pkg/test/server.go

```diff
@@ -78,3 +95,15 @@ type InstallStrategy struct {
 func (s *InstallStrategy) GetCRDs() []*extensionsv1beta1.CustomResourceDefinition {
 	return s.crds
 }
+
+// GetKubeCtl retrieve KubeCtl wrapper. It check Env and set default binary path.
+func (te *TestEnvironment) GetKubeCtl() *integration.KubeCtl {
+	if te.KubeCtl != nil {
+		return te.KubeCtl
+	}
+	kctl := te.ControlPlane.KubeCtl()
+	if os.Getenv(env_KubeCtl) == "" {
+		kctl.Path = bin_KubeCtl
+	}
+	return kctl
+}
```

Comment:
if we set the env for kubectl before starting the controlplane, then this method can simply return te.ControlPlane.Kubectl()